101-vm-from-user-image-and-datadisk/azuredeploy.json

```diff
@@ -0,0 +1,197 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vmName": {
+      "type": "string",
+      "metadata": {
+        "description": "This is the name of your virtual machine"
+      }
+    },
+    "userImageStorageAccountName": {
```

Comment:
It'd be easier to just take in a URI of the image from the user and have them enter less parameters